### DISCUSSION TITLE
positive TF motifs conceptual question

### DISCUSSION THREAD

**Discussion by AnjaliC4** [GreenleafLab/ArchR#1592] [Questions / Documentation]
Hi, from the chapter identifying positive TF regulators: "We can use all of this information to identify positive TF regulators. In the examples below, we consider positive regulators as those TFs whose correlation between motif and gene score (or gene expression) is greater than 0.5 with an adjusted p-value less than 0.01 and a maximum inter-cluster difference in deviation z-score that is in the top quartile."

I would like to understand conceptually why you suggest using "maximum inter-cluster difference in deviation z-score" ? From my understanding, the ones in top quartile would be the ones with the highest difference in their motif accessibility between clusters -is this understanding correct?  If so, could you please help me understand how does that relate to being a positive TF regulator. 

Thanks in advance for your explanation.

**Comment by AnjaliC4** [comment]
Hi @rcorces following up on this. Would really appreciate your input! Thanks again!

**Reply by rcorces** [reply]
I believe the logic is that if the motif deviations arent changing across clusters then there wont be anything to correlate to with gene expression.

**Reply by AnjaliC4** [reply]
Hi @rcorces, sorry to bug you again. I see your point but its still not super clear to me. As even within a cluster, a positive correlation of TF motif deviation with gene expression per-cell should be able to identify positive TF regulators for that cluster. I don't follow why TF motifs with high inter-cluster difference in their z-scores were also selected for this analysis - to me this would mean that those TFs had different activity between different clusters. Could you please clarify how does that relate to a positive TF?

**Reply by rcorces** [reply]
You are welcome to analyze your data however you think makes sense to you. The point of the analysis presented in the manual is to figure out, for example, which GATA factor is driving TF motif deviation. This type of analysis does not make sense to perform within cells of a single cluster in my opinion.
